dotcom-rendering/src/client/ga/ga.ts

```diff
@@ -82,13 +82,13 @@ export const sendPageView = (): void => {
 	const { GAData } = window.guardian;
 	const userCookie = getCookie({ name: 'GU_U', shouldMemoize: true });
 	const { ga } = window;
+	const isArticle = window.guardian.config.page.contentType === 'Article';
 
 	if (!ga) {
 		return;
 	}
-
 	ga(set, 'forceSSL', true);
-	ga(set, 'title', GAData.webTitle);
+	isArticle && ga(set, 'title', GAData.webTitle);
```

Comment:
Could we not couple the `contentType` and `GaData` by having a more explicit check of Data
```suggestion
	GAData.webTitle && ga(set, 'title', GAData.webTitle);
```